src/lib/layer.js

```diff
@@ -284,6 +290,16 @@ export default class Layer {
   }
 
   /**
+   * Returns the picking color of the object at 'selectedObjectIndex' prop.
+   * @return {Array} - the picking color or null if 'selectedObjectIndex is invalid.
+   */
+  selectedObjectPickingColor() {
+    if (this.props.selectedObjectIndex < 0) {
+      return null;
```

Comment:
This is not used by the apps but by draw-and-pick.js, so this is not really private, it is the same category as 'encodePickingColor()'.